The Mac Weekly/PostTableViewCell.swift

```diff
@@ -21,9 +21,25 @@ class PostTableViewCell: UITableViewCell {
     @IBOutlet weak var previewImageContainer: UIView!
     
     func loadPost(_ post: Post, shouldUseLargeImage: Bool = false) {
-        let formatter = DateFormatter()
-        formatter.dateFormat = defaultDateFormat
-        dateLabel.text = formatter.string(from: post.time)
+        
+        // Format the date nicely - Gabriel Brown
+        let postTimeInterval = -(post.time.timeIntervalSinceNow)  // Minus sign in front to flip the negative, since everything happened in the past
+        let timeUnit = TestableUtils.getTimeUnitToUse(timeInterval: postTimeInterval)
+        
+        if timeUnit != nil {
```

Comment:
Probably could be
```if let timeUnit = TestableUtils.getTimeUnitToUse(timeInterval: postTimeInterval) {```

Reply:
Not sure if this was an intentional decision or not, my suggestion is kind of a long line of code after all...

Reply:
That's some of the special swift syntactic sugar right? I think it has more to do with how I'm not used to swift yet haha. I'll be sure to switch that up.